app/components/EventItem/index.tsx

```diff
@@ -95,7 +153,60 @@ const EventItem = ({
           </Flex>
         </div>
       );
+    case 'compact':
+      return (
+        <div
+          style={{ borderColor: colorForEvent(event.eventType) }}
+          className={styles.eventItemCompact}
+        >
+          <Flex width="100%">
+            <Link to={`/events/${event.id}`}>
+              <h3 className={styles.eventItemTitle}>{event.title}</h3>
+            </Link>
+          </Flex>
+          <Flex width="100%" justifyContent="space-between">
+            <Flex width="72%">
+              <Flex className={styles.companyLogoCompact}>
+                {event.cover && (
+                  <Link to={`/events/${event.id}`}>
+                    <Image
+                      src={event.cover}
+                      placeholder={event.coverPlaceholder}
+                    />
+                  </Link>
+                )}
+              </Flex>
+            </Flex>
+            <Flex
+              display="flex"
+              justifyContent="flex-start"
+              column="true"
+              width="25%"
+            >
+              <Flex width="100%" justifyContent="flex-start">
+                <RegistrationIcon
+                  event={event}
+                  field={field}
+                  loggedIn={loggedIn}
+                />
+                <Attendance event={event} />
+              </Flex>
 
+              <TimeStamp event={event} field={field} loggedIn={loggedIn} />
+            </Flex>
+          </Flex>
+          {showTags && (
+            <Flex width="100%" justifyContent="flex-start">
+              <Flex wrap>
+                {event.tags.map((tag, index) => (
+                  <Tag key={index} tag={tag} />
+                ))}
+              </Flex>
+            </Flex>
```

Comment:
```suggestion
            <Flex wrap width="100%" justifyContent="flex-start">
              {event.tags.map((tag, index) => (
                <Tag key={index} tag={tag} />
              ))}
            </Flex>
```

Is this not the same thing?